test/helpers/ComponentMockeryTrait.php

```diff
@@ -263,5 +263,5 @@ protected function proxy($object): Proxy
      *
      * @return MockBuilder The mock builder.
      */
-    abstract function getMockBuilder($className);
-}
\ No newline at end of file
+    abstract function getMockBuilder($className): MockBuilder;
```

Comment:
This seems to be causing CI [builds][1] to fail.

[1]: https://travis-ci.com/Dhii/containers/jobs/279426491

Reply:
Indeed. Let's just remove it.